src/core/UnistylesRuntime.ts

```diff
@@ -1,14 +1,39 @@
 import { ScreenOrientation, UnistylesError } from '../common'
-import type { UnistylesBridge, UnistylesPlugin } from '../types'
+import type { Color, UnistylesBridge, UnistylesPlugin } from '../types'
 import type { UnistylesThemes } from '../global'
 import type { UnistyleRegistry } from './UnistyleRegistry'
+import { parseColor } from '../utils'
 
 /**
  * Utility to interact with the Unistyles during runtime
  */
 export class UnistylesRuntime {
     constructor(private unistylesBridge: UnistylesBridge, private unistylesRegistry: UnistyleRegistry) {}
 
+    /**
+     * Get the mini runtime injected to creteStyleSheet
+     * @returns - The mini runtime
+     */
+    public get miniRuntime() {
```

Comment:
Love this change by the way. 👏

Reply:
Thanks!
Btw. Im always looking for your ideas like with font scale and pixel ratio